rest-api/src/main/resources/definitions/alert.yml

```diff
@@ -0,0 +1,35 @@
+description: An alert to be displayed.
+type: object
+required:
+    - id
+    - createdOn
+    - participant
+    - category
+properties:
+    id:
+        type: string
+        description: The unique ID of this alert.
+        readOnly: true
+        x-nullable: false
```

Comment:
As per our previous discussion, the x-nullable attribute does nothing. Moving forward, we should stop including it in our JavaSDK changes.